### PR TITLE
Add post to portfolio

### PR DESCRIPTION
**Pull request by antoniolinhart**
- Add form to use post from html to DataServlet
- Add 'fetch comments' button to get a list of comments added to DataServlet

### CODE REVIEW

File: portfolio/src/main/webapp/index.html [hunk 1]
```diff
@@ -102,9 +102,16 @@ <h1><span class="text-emphasis">Antonio</span>'s Portfolio</h1>
         </div>
       </div>
       <div id="comments">
+        <form action="/data" method="POST">
+          <p>Enter a comment:</p>
```
Comment (geoffbass) [inline portfolio/src/main/webapp/index.html:106]: This can probably be a `<label>` for the textarea

File: portfolio/src/main/webapp/index.html [hunk 2]
```diff
@@ -102,9 +102,16 @@ <h1><span class="text-emphasis">Antonio</span>'s Portfolio</h1>
...
+          <textarea name="text-input"></textarea>
+          <br/><br/>
```
Comment (geoffbass) [inline portfolio/src/main/webapp/index.html:108]: nit: Omit the `/` in void elements (https://google.github.io/styleguide/htmlcssguide.html#Document_Type)

You could also probably omit the `<br>`s altogether and get the layout you want using CSS. `display` should help.

File: portfolio/src/main/webapp/index.html (hunk 1 above)
Comment (antoniolinhart) [reply portfolio/src/main/webapp/index.html:106]: Didn't know about labels before, thank you!

File: portfolio/src/main/webapp/index.html (hunk 2 above)
Comment (antoniolinhart) [reply portfolio/src/main/webapp/index.html:108]: Changed it to use `display` tags instead of `br`!

**Pull request merged**